docs/blogs/IntroGatewayAPI/IntroGatewayAPI.md

```diff
@@ -0,0 +1,165 @@
+and is offered as a Developer Preview feature in OpenShift 4.12.
+
+OpenShift has had a cluster ingress solution since its first release, in order to support self-service of cluster ingress
+by developers and namespace administrators.  Kubernetes later added the Ingress API specification, with many backing
+implementations (including OpenShift).  However, the Ingress API left a lot to be specified by the implementation, making
+it difficult to develop portable Ingress objects across implementations.  To address this, the Ingress v2 working group
+was started, and Red Hat has been involved throughout.  This group evolved into what is today the
+[Kubernetes Gateway API working group](https://docs.openshift.com/container-platform/4.11/networking/configuring_ingress_cluster_traffic/overview-traffic.html)
+, which continues to garner robust community support.
+
+![Istio Gateway API compared to OpenShift HAProxy Architecture](images/IstioVsOpenShift.png)
+
+### Gateway API Resources
+The basic building blocks of the Gateway API are resources called HTTPRoute, Gateway, and GatewayClass.
+
+ Resource | Definition | Relationships 
+--------------|------------------------------------------------------------------------------------------|--------------
+ HTTPRoute | Specifies rules for mapping requests from a Gateway to Kubernetes Services, especially for multiplexing HTTP or terminated HTTPS connections. | References Gateway/s 
+ Gateway | Describes how traffic can be translated to Services within the cluster, e.g. a specific load balancer config. | Implements GatewayClass
+ GatewayClass | Defines a set of Gateways that share a common configuration and behavior. |
+
+Shown below is an example of a HTTPRoute named `my-route` that is configured to attach only to the gateway named `example-gateway`
+in the namespace `example-ns1`, and will not attach to any other Gateways.  Note that `parentRefs` refer to the gateway and
+`backendRefs` refer to service endpoints.
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: my-route
+  namespace: example-ns2
+spec:
+  parentRefs:
+  - kind: Gateway
+    name: example-gateway
+    namespace: example-ns1
+  rules:
+  - backendRefs:
+    - name: example-service
+      port: 8080
+```
+The next example shows the Gateway `example-gateway`, configured to allow the `my-route` HTTPRoute to attach.  Note the
+reference to `gatewayClassName`.
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: example-gateway
+  namespace: example-ns1
+spec:
+  gatewayClassName: example-lb
+  listeners:
+  - name: prod-web
+    port: 80
+    protocol: HTTP
+    allowedRoutes:
+      kinds: 
+        - kind: HTTPRoute
+      namespaces:
+        from: Selector
+        selector:
+          matchLabels:
+            # This label is added automatically as of K8s 1.22
+            # to all namespaces
+            kubernetes.io/metadata.name: gateway-api-example-ns2
```

Comment:
This needs to select the namespace from the route above, right?  
```suggestion
            kubernetes.io/metadata.name: example-ns2
```

Reply:
Thank you!  This has been wrong from the beginning.